frontend/src/pages/Home.jsx

```diff
@@ -31,34 +52,355 @@ class Home extends Component {
           </Typography>
           <Button 
             variant="contained" 
-            color="primary"
             style = {{
-              marginRight: 10
+              marginRight: 10,
+              color: "#0F1111",
+              background: "#f0c14b",
+              fontWeight: 600
+            }}
+            onClick = {()=>{
+              this.props.setLogin_dialog(true)
+              this.props.setRegister_dialog(false)
             }}
           >
-            Sign in 
-          </Button>
-          <Button variant="contained" color="primary">
-            Register
+            Log In
           </Button>
+          
+          {/* This is for the login dialog */}
```

Comment:
Is this comment supposed to be inside curly braces?

Reply:
Ah, is this a React Native thing?

Reply:
Yeah. This is a ReactJS thing. ReactJS uses JSX syntax for rendering. This allows dev to write JavaScript into a HTML template. In order to use JS, we need to put a curly bracket {} surrounding the JS.
- Reference: https://reactjs.org/docs/introducing-jsx.html

Reply:
In order to comment something in JSX code, u need to wrap it in {/* some comment here*/}.
And u can add multiple lines between the start of comment:"{/*" and end of comment: "*/}"
JSX code is the part that wrapped in "render" function, the part in return ()